product_docs/docs/biganimal/release/using_cluster/01_postgres_access.mdx

```diff
@@ -11,9 +11,9 @@ psql -W "postgres://edb_admin@xxxxxxxxx.xxxxx.biganimal.io:5432/edb_admin?sslmod
 ```
 ## Notes on the edb_admin role
 
-- The `edb_admin` role does not have superuser priviledges by default. Contact [Support](../overview/06_support) to request superuser priviledges for `edb_admin`.
+- The `edb_admin` role does not have superuser priviledges by default. Contact [Support](../overview/06_support) to request superuser priviledges for `edb_admin`. If you request superuser privileges, you **must** take care to limit the number of connections used by superusers to avoid degraded service and/or compromising availability.
 
-- superuser setting changes are not retained after a restart. To avoid issues, do not run the system out of superuser connections.
+- Changes to system configuration (GUCs) made by `edb_admin` or other Postgres users will not be persisted though a reboot or maintenance. Use the BigAnimal portal to modify system configuration.
```

Comment:
```suggestion
- Changes to system configuration (GUCs) made by edb_admin or other Postgres users are not persisted though a reboot or maintenance. Use the BigAnimal portal to modify system configuration.
```

Reply:
We are updating our conventions to not use a font treatment for user names. We also avoid future case.